scripts/issue/354/upgrade_remaining_setts_pause.py

```diff
@@ -0,0 +1,315 @@
+    "bslpWbtcEth",
+]
+
+SETTV4_KEYS = [
+    "bcrvHBTC",
+    "bcrvPBTC",
+    "bcrvOBTC",
+    "bcrvBBTC",
+    "bcrvIbBTC",
+    "bcrvTricrypto",
+    "bcrvTricrypto2",
+    "bcvxCRV",
+    "bCVX",
+    "bveCVX",
+    "bimBTC",
+    "bFpMbtcHbtc",
+    "bbveCVX-CVX-f",
+    "bcrvBadger",
+]
+
+ACTIVE_V4_KEYS = [
+    "bveCVX",
+    "bcrvIbBTC",
+    "bcrvBadger",
+    "bbveCVX-CVX-f",
+    "bcvxCRV",
+    "bcrvTricrypto2",
+]
+
+SPECIAL_SETTS = [
+    "bDIGG",
+    "byvWBTC",
+    "remBADGER",
+    "remDIGG",
+]
+
+UPGRADE_SIG = "upgrade(address,address)"
+SET_STRATEGIST_SIG = "setStrategist(address)"
+
+
+def main(queue="true", simulation="false"):
+    if queue == "true":
+        for key, address in vaults.items():
+            address = web3.toChecksumAddress(address)
+            if key in SETTV1_KEYS:
+                if get_implementation(address) != settV1h_logic:
+                    console.print(
+                        f"[green]Queueing upgrade on timelock for {key} to SettV1h.sol[/green]"
+                    )
+                    safe.badger.queue_timelock(
+                        target_addr=DEV_PROXY,
+                        signature=UPGRADE_SIG,
+                        data=encode_abi(
+                            ["address", "address"],
+                            [address, settV1h_logic],
+                        ),
+                        dump_dir="data/badger/timelock/upgrade_remaining_setts/",
+                        delay_in_days=4,
+                    )
+                else:
+                    console.print(f"[green]{key} has already been upgraded[/green]")
+
+            elif key in SETTV1_1_KEYS:
+                if get_implementation(address) != OLD_V1_1H_LOGIC:
+                    console.print(
+                        f"[green]Queueing upgrade on timelock for {key} to SettV1_1h.sol[/green]"
+                    )
+                    safe.badger.queue_timelock(
+                        target_addr=DEV_PROXY,
+                        signature=UPGRADE_SIG,
+                        data=encode_abi(
+                            ["address", "address"],
+                            [address, settV1_1h_logic],
+                        ),
+                        dump_dir="data/badger/timelock/upgrade_remaining_setts/",
+                        delay_in_days=4,
+                    )
+                    console.print(
+                        f"[green]Queueing setStrategist on timelock for {key} to tech ops {TECH_OPS}[/green]"
+                    )
+                    safe.badger.queue_timelock(
+                        target_addr=DEV_PROXY,
+                        signature=SET_STRATEGIST_SIG,
+                        data=encode_abi(
+                            ["address"],
+                            [TECH_OPS],
+                        ),
+                        dump_dir="data/badger/timelock/upgrade_remaining_setts/",
+                        delay_in_days=4,
+                    )
```

Comment:
The setStrategy function can't be called from the Timelock, it will revert on execution. It should instead be called from the devMulti and can be done atomically along with the tx executions in the second part. This is because `setStrategy()` is a `_onlyGovernance` function and governance on all Setts should be the devMulti and not the Timelock (should be good to confirm this via the script).
![image](https://user-images.githubusercontent.com/25463788/167877228-351ff233-ab92-4155-b605-e587221952f0.png)

Function: https://etherscan.io/address/0x4Da27cD2AE34a9E1776Ed01747A071C17Fa0b2Cf#code#L715